pkg/objstore/providers/s3/config.go

```diff
@@ -108,6 +121,14 @@ func (cfg *Config) Validate() error {
 		return errUnsupportedSignatureVersion
 	}
 
+	if cfg.ForcePathStyle && cfg.BucketLookupType == VirtualHostedStyleLookup {
```

Comment:
Nit: Might be slightly more failsafe, once there is another lookup type added (I hope not 😆 )

```suggestion
	if cfg.ForcePathStyle && cfg.BucketLookupType != AutoLookup && cfg.BucketLookupType != PathStyleLookup {
```

Reply:
> Nit: Might be slightly more failsafe, once there is another lookup type added (I hope not 😆 )

Fixed